frontend/src/locale/fr.yml

```
@@ -0,0 +1,1471 @@
+---
+#-------------------------------------------------------------#
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)